README.md

```diff
@@ -61,7 +61,10 @@ The Paragon CLI (Command Line Interface) is a tool that provides various utility
 
 ### Available Commands
 
-- `paragon install-theme [theme]`: Installs the specific @edx/brand package.
+- `paragon install-theme [theme]`: Installs the specific [@edx/brand](https://github.com/edx/brand-edx.org) package.
```

Comment:
This should reference the openedx brand package instead of the edx.org one

https://github.com/openedx/brand-openedx

Reply:
Good point, updated, thanks.